dev/import-beats/fields.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+func loadDatasetFields(modulePath, datasetName string) ([]byte, error) {
```

Comment:
Not sure if I would concatenate the fields.yml together. The package manager can support multiple `*.yml` files. Instead you could keep them separate but you need to add a top block to each dataset with the right "prefix". Keeping them separate has a few benefits I think:

* The shared part could become a component which is reused in elasticsearch templates v2
* During development I think it helps the dev to differentiate which are global fields and which ones are not. Like this if he changes a global field, he knows to check all datasets.

It could be argued, that the global fields should be on the package level but lets skip this for now as EPM does not support it.

Reply:
Do you have any preferences regarding naming?

in every `module/dataset/foo/fields/`:
`module-fields.yml`
`fields.yml`

Reply:
No preference. The above LGTM.

Reply:
I'll address this comment in https://github.com/elastic/package-registry/pull/240